src/main/java/com/nekromant/telegram/service/ResumeAnalysisRequestService.java

```diff
@@ -1,17 +1,95 @@
+    @Autowired
+    private MentoringReviewBot mentoringReviewBot;
+    @Autowired
+    private LifePayFeign lifePayFeign;
+    @Autowired
+    private TelegramFeign telegramFeign;
+    @Autowired
+    private CVAnalProp props;
 
-    public void save(ResumeAnalysisRequest resumeAnalysisRequest) {
-        resumeAnalysisRequestRepository.save(resumeAnalysisRequest);
+
+    public void save(byte[] bytes, String tgName, String phone) {
+        ResumeAnalysisRequest resumeAnalysisRequest = new ResumeAnalysisRequest();
+        resumeAnalysisRequest.setCVPdf(bytes);
+        resumeAnalysisRequest.setTgName(tgName);
+        resumeAnalysisRequest.setPhone(phone);
+
+        String description = "Оплата за разбор резюме по договору публичной оферты";
+        ChequeDTO chequeDTO = new ChequeDTO(props.getLogin(), props.getApikey(), props.getAmount(), description, phone, props.getMethod());
+
+        try {
+            ResumeAnalysisRequestService.log.info("Sending request to LifePay" + chequeDTO);
+            LifePayResponseDTO lifePayResponse = new Gson().fromJson(lifePayFeign.payCheque(chequeDTO).getBody(), LifePayResponseDTO.class);
+            ResumeAnalysisRequestService.log.info("LifePay response: " + lifePayResponse);
+            PaymentDetails paymentDetails = new PaymentDetails();
+            paymentDetails.setNumber(lifePayResponse.getData().getNumber());
+            paymentDetails.setStatus(PayStatus.UNREDEEMED.get());
+            paymentDetailsRepository.save(paymentDetails);
+            ResumeAnalysisRequestService.log.info("Unredeemed payment created: " + paymentDetails);
+
+            resumeAnalysisRequest.setLifePayNumber(lifePayResponse.getData().getNumber());
+            resumeAnalysisRequestRepository.save(resumeAnalysisRequest);
+            ResumeAnalysisRequestService.log.info("New resume analysis request created: " + resumeAnalysisRequest);
+        } catch (Exception e) {
+            log(e.getMessage());
+        }
     }
 
+    public void sendCVToMentorForAnalysisOrReject(PaymentDetails paymentDetails) {
+        paymentDetailsRepository.deleteByNumber(paymentDetails.getNumber());
+        paymentDetailsRepository.save(paymentDetails);
+
+        if (paymentDetails.getStatus().equals(PayStatus.FAIL.get())) {
+            ResumeAnalysisRequestService.log.info("Payment failed: " + paymentDetails);
+            return;
+        }
+
+        ResumeAnalysisRequestService.log.info("Payment details have been redeemed:" + paymentDetails);
+
+        byte[] CV_bytes = resumeAnalysisRequestRepository.findByLifePayNumber(paymentDetails.getNumber()).getCVPdf();
+        FormData formData = new FormData(MediaType.MULTIPART_FORM_DATA, "document", CV_bytes);
+        String receiverId = userInfoService.getUserInfo(props.getReceiverName()).getChatId().toString();
+        telegramFeign.sendDocument(formData, receiverId);
+
+        String text = "Зарегистрирован и оплачен заказ на разбор резюме " +
```

Comment:
строковую константу бы вынес в константу, а для формирования строки вместо конкатенации опрятнее выглядит String.format(...) что-то типа
```suggestion
        String text = String.format(RESPONSE_FOR_RESUME_PROJARKA, paymentDetails.getNumber(), paymentDetails.getPhone(), ...);
```

Reply:
done